result_set.go

```diff
@@ -357,8 +357,12 @@ func (res ResultSet) scanRow(row *nebula.Row, colNames []string, t reflect.Type)
 		rowVal := rowVals[cIdx]
 
 		switch f.Type.Kind() {
+		case reflect.Int:
+			val.Field(fIdx).SetInt(rowVal.GetIVal())
 		case reflect.Int64:
 			val.Field(fIdx).SetInt(rowVal.GetIVal())
+		case reflect.Float64:
+			val.Field(fIdx).SetFloat(rowVal.GetFVal())
```

Comment:
don't we need to process bool/float32/int8/int16/date/datetime/time/duration/?

Reply:
we need, but I will open another PR to do this.
Also we need enhance/refactor the tests.
this is just make the example works.